### DISCUSSION TITLE
State of integration

### DISCUSSION THREAD

**Discussion by dylan09** [B5r1oJ0A9G/teufel_raumfeld#19] [Q&A]
Hello,

how is the state of the integration? Is development still active?
I have tested current release. And I encountered a few errors and problems. So I decided to uninstall for now.
But was highly interested in this integration. Maybe I could help and support further development.

I have one Raumfeld Base, and a few Raumfeld Speaker (L, M and S) installed.

**Comment by B5r1oJ0A9G** [comment]
Hello,

Thanks for the question!

I am not actively developing it at the moment. However, it is not discontinued either. The reason for creating this implementation was a private need - which is currently being fully met. As you may have seen, there have been some fixes and improvements based on user feedback and contributions.

Anything that I could or can reproduce myself without changes to the infrastructure is usually fixed in a reasonable amount of time (as much as my personal life allows). For everything else, I rely on user feedback or testing. If this is not possible, for whatever reason, the development in question will probably be put on hold.

Support is always appreciated, of course! :)
